olad/www/new/index.html

```diff
@@ -54,6 +54,15 @@
  </div>
  <!-- /.container-fluid -->
 </nav>
+<noscript>
+  <br>
+  <p align="center">
+    <strong>This interface requires JavaScript to function. If you
```

Comment:
If I'm being really pedantic, it would be nice if the indentation was consistent in each code block, you've got some on the would line here, but not on the others.